File: python/002/002.py
print("What a nice dinner! Time to pay the bill.")
bill_amount = float(input("Enter the total bill amount: $"))
recommended_tip = bill_amount * 0.15
print(f"A recommended tip (15%) would be: ${recommended_tip:.2f}")
tip_amount = float(input("Enter the tip amount you would like to give: $"))
total_amount = bill_amount + tip_amount
print(f"The total amount to be paid is: ${total_amount:.2f}")

File: go/002/002.go
package main

import (
	"fmt"
	
)

func main() {
	fmt.Println("Hello! Did you enjoy the dinner?")
	
	var total float64 = 0.0
	var tip float64 = 0.0

	fmt.Println("Time to pay the bill. The total is: ")

	fmt.Scanf("%f\n", &total)

	fmt.Print("The recommended tip is 15%, which is: ")
	fmt.Printf("%.2f\n", total*0.15)
	fmt.Println("How much would you like to tip?")

	fmt.Scanf("%f\n", &tip)

	fmt.Printf("The total amount to be paid is: %.2f\n", total+tip)
}